packages/react-components/react-jsx-runtime/src/utils/types.ts

```diff
@@ -0,0 +1,9 @@
+import type * as React from 'react';
+
+export type JSX = <P extends {}>(
```

Comment:
```suggestion
export type JSXFactory = <P extends {}>(
```

nit: Feel free to pick another option, but let's make this clearer